.circleci/config.yml

```diff
@@ -27,7 +27,6 @@ jobs:
               name: Pull Docker images
               command: |
                 docker pull kidsfirstdrc/torinox:torinox.r4-latest
-                docker pull kidsfirstdrc/fhir-ig-publisher:latest
```

Comment:
`scripts/run_publisher.sh` is set to being always puling `'kidsfirstdrc/fhir-ig-publisher:latest'` whereas CircleCI pulls a different image?

Reply:
CircleCI was pulling two images - 1) For torinox a CLI tool for converting XML to JSON 2) IG publisher. I removed the pull for 2 because now the run_publisher.sh script does it